pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -224,6 +232,14 @@ type TLSConfig struct {
 	ServerName string `json:"serverName,omitempty"`
 	// Disable target certificate validation.
 	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty"`
+	// Minimum TLS version. Accepted values: TLS10 (TLS 1.0), TLS11 (TLS 1.1), TLS12 (TLS 1.2), TLS13 (TLS 1.3).
```

Comment:
Do we want to allow TLS 1.0 and 1.1, even though those are deprecated? Any advantage to doing so?

Reply:
I added them solely because Prometheus supports it. Note, those deprecated versions are disabled by default.